examples/offline.js

```diff
@@ -28,10 +28,11 @@ app.offline.module = angular.module('app', [
 
 /**
  * @param {ngeoFeatureOverlayMgr} ngeoFeatureOverlayMgr ngeo feature overlay manager service.
+ * @param {ngeoNetworkStatus} ngeoNetworkStatus ngeo network status service.
  * @constructor
  * @ngInject
  */
-app.offline.MainController = function(ngeoFeatureOverlayMgr) {
+exports.MainController = function(ngeoFeatureOverlayMgr, ngeoNetworkStatus) {
```

Comment:
Are you sure, my code is more ready for es6 module no ?
It comes from here at least

Reply:
For the examples, that ~true. But I don't now what will happen with the transformation. I prefers to use the standard case, I think that will be easier to transform.

For the src code, we can't. Because the  app (and so luxembourg) still work without webpack.